templates/repo/settings/lfs_file.tmpl

```diff
@@ -19,7 +19,7 @@
 					{{else if .IsPlainText}}
 						<pre>{{if .FileContent}}{{.FileContent | Safe}}{{end}}</pre>
 					{{else if not .IsTextFile}}
-						<div class="view-raw ui center">
+						<div class="view-raw ui center{{if .IsPDFFile}} gt-p-0{{end}}">
```

Comment:
I haven't quite understood this change here and in the next file.
Why only when it is a PDF file?

Reply:
Could maybe done for other types as well, I need to check.

Reply:
Removed the margin from CSS and did a few more cleanups to view raw.

audio:

<img width="1207" alt="Screenshot 2023-05-28 at 15 50 56" src="https://github.com/go-gitea/gitea/assets/115237/c8f2e97e-0dbf-4ce1-82bd-6777590761d7">

video:
<img width="1200" alt="Screenshot 2023-05-28 at 15 51 07" src="https://github.com/go-gitea/gitea/assets/115237/3476c32e-23e7-474c-871c-3a71641e6468">

image:
<img width="1207" alt="Screenshot 2023-05-28 at 15 51 46" src="https://github.com/go-gitea/gitea/assets/115237/74d0b450-1234-41dd-9ecf-3befad5a4999">

pdf corner:
<img width="110" alt="Screenshot 2023-05-28 at 15 57 48" src="https://github.com/go-gitea/gitea/assets/115237/fd3321de-5c88-4bdf-a207-9d1f3e21de15">